src/chains.ts

```diff
@@ -208,5 +236,9 @@ export const L1_STANDARD_BRIDGE_INFORMATION: Record<
       l2Chain: 'lisk-sepolia',
       l1StandardBridgeAddress: '0x1Fb30e446eA791cd1f011675E5F3f5311b70faF5',
     },
+    {
+      l2Chain: 'metall2-sepolia',
+      l1StandardBridgeAddress: '0xFA0EBa40F30338B50A08613Af2216f914Ae8a7B4',
```

Comment:
same here https://github.com/ethereum-optimism/superchain-registry/blob/10ff8b6e7b57cbd894d333d13c59c08bb3970cac/superchain/configs/sepolia/metal.toml#L45

Reply:
@nitaliano I've fixed these issues. Thank you.